terasoluna-gfw-parent/pom.xml

```diff
@@ -211,6 +211,7 @@
         <groupId>com.github.dozermapper</groupId>
         <artifactId>dozer-core</artifactId>
         <version>${dozer.version}</version>
+        <classifier>jar-with-dependencies</classifier>
```

Comment:
- [ ] Why required `classifier=jar-with-dependencies`? Is there no problem even without it?